### ISSUE TITLE
Grafana 10.1.0: RPM - Invalid/different version string for current version

### ISSUE THREAD

**Issue by sc-amaus** [grafana/grafana#73731]
### What happened?

Grafana 10.1.0 has a different version string / schema than all previous releases:

```
rpm -qpi grafana-10.1.0-1.x86_64.rpm | grep ^Version
Version     : v10.1.0
```

This seems to be a problem for YUM/DNF as it don't recognize the update for this package.

### What did you expect to happen?

Consistent version string like all previous releases withot "v" before the version number

```
rpm -qpi grafana-10.0.3-1.x86_64.rpm  | grep ^Version
Version     : 10.0.3
```

### Did this work before?

Yes

### How do we reproduce it?

1. Download the RHEL RPM from https://grafana.com/grafana/download?edition=oss
2. Run `rpm -qpi grafana-10.1.0-1.x86_64.rpm`

### Is the bug inside a dashboard panel?

no

### Environment (with versions)?

Grafana: 10.1.0
OS: Redhat Enterprise Linux 8, x86_64
Browser: -

### Grafana platform?

A package manager (APT, YUM, BREW, etc.)

**Comment by usmangt** [comment]
Hi @sc-amaus,

Thanks for opening this issue.

I will forward this to our Engineers so that they can have a look and might be able to tell more details about it.

**Comment by RuiSantos76** [comment]
Just to let you know, it's also happening on OpenSUSE Leap 15.5:

rpm -Uhv grafana-enterprise-10.0.4-1.x86_64.rpm --test --nodeps
Preparing...                          ################################# [100%]
        package grafana-enterprise-9.5.8-1.x86_64 (which is newer than grafana-enterprise-v10.0.4-1.x86_64) is already installed

**Comment by kminehart** [comment]
This should be resolved now as of `10.1.1`.

I think in the near future we'll also be releasing 10.0.4 with the same packaging tool.

```
[root@1a1ef6cca6d9 /]# wget https://dl.grafana.com/oss/release/grafana-10.1.1-1.x86_64.rpm && rpm -qpi *.rpm  | grep ^Version
warning: grafana-10.1.1-1.x86_64.rpm: Header V4 RSA/SHA512 Signature, key ID 10458545: NOKEY
Version     : 10.1.1
```

**Comment by sc-amaus** [comment]
Confirmed, updates are deployed on all instances.
Thank you very much ;)
